src/main/webapp/index.html

```diff
@@ -1,73 +1,75 @@
-            <a class="link" href="#">Favorite Notes</a>
-            <a class="link" href="#">Logout</a>
-          </div>
-        </div>
-      </div>
-    </div>
-    <!-- <nav class="zone blue sticky"> -->
-    <!--   <ul class="main-nav"> -->
-    <!--     <li> -->
-    <!--       <a href="">starfish</a> -->
-    <!--     </li> -->
-    <!--     <div class="push"> -->
-    <!--       <li> -->
-    <!--         <a href="">PostANote</a> -->
-    <!--       </li> -->
-    <!--       <li> -->
-    <!--         <a href=""><img id="user-profile" src="assets/user.svg"></a> -->
-    <!--       </li> -->
-    <!--     </div> -->
-    <!--   </ul> -->
-    <!-- </nav> -->
-    <main class="blue">
-      <h1 class="title">starfish</h1>
-      <h3 class="subtitle">Share and find the best course notes. Completely free.</h3>
-      <form target="_blank">
-        <label for="topic">Find by Topic:</label><br>
-        <input type="text" name="topic"><br>
-        <div class="inline">
-          <span class="inline-form-el">
-            <label for="school">School:</label><br>
-            <input type="text" name="school"><br>
-          </span>
-          <span class="inline-form-el">
-            <label for="course">Course:</label><br>
-            <input type="text" name="course"><br>
-          </span>
-        </div>
-        <input type="submit" value="Submit" onclick="window.location.href='./searchresults.html'">
-      </form>
-      <img id="laptop" src="assets/laptop-min.png">
-    </main>
-  </body>
-</html>
+<!-- Bunch of Stylesheets for Vuetify -->
+<link href="https://fonts.googleapis.com/css?family=Roboto:100,300,400,500,700,900" rel="stylesheet">
+<link href="https://cdn.jsdelivr.net/npm/@mdi/font@5.x/css/materialdesignicons.min.css" rel="stylesheet">
+<link href="https://cdn.jsdelivr.net/npm/vuetify@2.x/dist/vuetify.min.css" rel="stylesheet">
+<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no, minimal-ui">
+<!-- Necessary for vue to work, but id can be anything as long as reflected in script below -->
+<div id="app">
+  <!-- Necessary for vuetify to work -->
+  <v-app>
+    <navbar></navbar>
+    <search-bar @searchresult="updateSearchResult"></search-bar>
+    <mock-search @searchresult="updateSearchResult"></mock-search>
+    <note-grid v-slot="{ filters }">
+      <note-grid-collection :note-data="searchResult"
+                            header="Most Favorited"
+                            :filters="filters"
+                            :compare-func="descFavorites">
+      </note-grid-collection>
+      <note-grid-collection :note-data="searchResult"
+                            header="Recently Posted"
+                            :filters="filters"
+                            :compare-func="descDate">
+      </note-grid-collection>
+      <note-grid-collection :note-data="searchResult"
+                            header="Hottest Notes"
```

Comment:
Is the only thing distinguishing this from "Most favorited" that it's a subset of the other collection with some date restriction?  From a UX perspective, I'm not sure having both would really be necessary...?

Reply:
Good point! The idea was to highlight notes that have been the most active recently (regardless of age), but that will probably require a lot of back end work so this is just a placeholder, really. I can remove or comment it out with an explanation.